x/pki/handler_test.go

```diff
@@ -1942,6 +1955,716 @@ func TestHandler_ApproveX509RootCert_FourApprovalsAreNeeded_FiveTrustees(t *test
 	require.Equal(t, testconstants.RootSerialNumber, approvedCertificate.SerialNumber)
 }
 
+func TestHandler_AddPkiRevocationDistributionPoint_PAASenderNotVendor(t *testing.T) {
+	setup := Setup(t)
+
+	addPkiRevocationDistributionPoint := types.MsgAddPkiRevocationDistributionPoint{
+		Signer:               setup.Trustee1.String(),
+		Vid:                  65521,
+		IsPAA:                true,
+		Pid:                  8,
+		CrlSignerCertificate: testconstants.PAACertWithNumericVid,
+		Label:                "label",
+		DataURL:              testconstants.DataURL,
+		IssuerSubjectKeyID:   testconstants.SubjectKeyIDWithoutColons,
+		RevocationType:       1,
+	}
+	_, err := setup.Handler(setup.Ctx, &addPkiRevocationDistributionPoint)
+	require.ErrorIs(t, err, sdkerrors.ErrUnauthorized)
+}
+
```

Comment:
Please make sure that both key-value storages (main one and by-issuer-subject-key-id) are updated after successful additions.